docs/01-app/03-api-reference/04-functions/expirePath.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: expirePath
+description: API Reference for the expirePath function.
```

Comment:
Might want to [mark these as canary](https://vercel.slack.com/archives/C06DGJ420TZ/p1730990131156869?thread_ts=1730208016.298569&cid=C06DGJ420TZ) until the next minor release? 

```suggestion
description: API Reference for the expirePath function.
version: canary
```

Reply:
Won't these only show on the canary docs version until we cut a stable release?